src/cmd/projects.js

```diff
@@ -75,10 +74,39 @@ export class Projects {
 		return os.homedir();
 	}
 
-	// todo - documnents folder?
+	windowsDocumentsFolder() {
+		const winreg = require('winreg');
+		const regKey = winreg({
+			hive: winreg.HKCU,                                        // open registry hive HKEY_CURRENT_USER
+			key:  '\\Software\\Microsoft\\Windows\\CurrentVersion\\Explorer\\Shell Folders' // key containing user folder paths
+		});
+		let documents;
+
+		// list "Documents" folder path
+		regKey.values((err, items) => {
+			if (err) {
+				console.log('ERROR: '+err);
+			} else {
+				for (let i=0; i<items.length; i++) {
+					if (items[i].name === 'Personal') {
+						documents = items[i].value;
+					}
+				}
+			}
+		});
+		return documents;
```

Comment:
`regKey.values` is async since it starts an external process so this has to return a promise :disappointed: